### PR TITLE
Make it easier to figure out where the current G43 values are stored.

### PR DESCRIPTION
**Pull request by petterreinholdtsen**

### CODE REVIEW

File: docs/src/gcode/g-code.adoc [hunk 1]
```diff
@@ -1392,6 +1392,9 @@ G43 <H->
 
 'G43 Hn' uses the offset for tool n.
 
+The active tool length compensation values are stored in the numbered
+parameters '5401-5409'.
```
Comment (SebKuzminsky) [inline docs/src/gcode/g-code.adoc:1396]: I think there should be no newlines in this hunk, to support translations.

**Review by SebKuzminsky — CHANGES_REQUESTED** [review]

**Comment by petterreinholdtsen** [comment]
[Sebastian Kuzminsky]
> +The active tool length compensation values are stored in the numbered
> +parameters '5401-5409'.
>
> I think there should be no newlines in this hunk, to support
> translations.

The msgid generated by po4a by running 'make -C src pofiles' for this
string look like this:

msgid ""
"The active tool length compensation values are stored in the numbered "
"parameters '5401-5409'."

This support translations just fine.

-- 
Happy hacking
Petter Reinholdtsen

File: docs/src/gcode/g-code.adoc (hunk 1 above)
Comment (hansu) [reply docs/src/gcode/g-code.adoc:1396]: @smoe what is your experience here?

**Comment by smoe** [comment]
I have to paddle back from (a good part of) my past corrections - when the text is at the top level of the document, then po4a and weblate seem to indeed work fine and present this without a newline. Weblate is doing some maintenance at the moment, so I cannot verify. I suggest to accept as it is and if it fails then Petter is the one to communicate this to the po4a folks.

File: docs/src/gcode/g-code.adoc (hunk 1 above)
Comment (zz912) [reply docs/src/gcode/g-code.adoc:1396]: > I think there should be no newlines in this hunk, to support translations.

My view of the matter is that. That translations should not hinder improvement.

**Pull request merged**